web/src/lib/static/mockHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import fs from "fs";
+
+export const mockReadDirReturn = (value: string[], mockedFs: jest.Mocked<typeof fs>): void => {
```

Comment:
[pebble/dust] can we parameterize these arguments so the function can be called something along the lines of 
```
mockReadDirReturnOnce({
  mockedFs, 
  values/returnValues: ["license1.md", "license2.md"]
});
```
to improve readability?